From 13beaff6f3c5fe10381b5795163867522435bbcb Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Fri, 23 Feb 2018 16:56:46 +0100
Subject: [PATCH] Include top-icons in classic session

---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 955b5ee..df2be33 100644
--- a/meson.build
+++ b/meson.build
@@ -38,6 +38,7 @@ classic_extensions = [
   'apps-menu',
   'places-menu',
   'launch-new-instance',
+  'top-icons',
   'window-list'
 ]
 
@@ -59,7 +60,6 @@ all_extensions += [
   'no-hot-corner',
   'panel-favorites',
   'systemMonitor',
-  'top-icons',
   'updates-dialog',
   'user-theme',
   'window-grouper'
-- 
2.21.0

